src/components/TodayRoutine/TodayRoutine.js

```diff
@@ -0,0 +1,170 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import RoutineThumbNail from '../RoutineThumbNail/RoutineThumbNail';
+
+const TodayRoutine = () => {
+  const [TodayRoutineData, setTodayRoutineData] = useState({});
+  const userNickName = localStorage.getItem('userNickname');
+
+  useEffect(() => {
+    fetch('/data/soonwoo.json', {
+      method: 'GET',
+    })
+      .then(response => {
+        return response.json();
+      })
+      .then(result => {
+        setTodayRoutineData(result.data);
+      });
+  }, []);
+
+  if (Object.keys(TodayRoutineData).length <= 0) return null;
+
+  const sliceData = exercises.slice(0, 3);
```

Comment:
답변해주신 대로라면, 3을 하드코딩으로 작성하는 것보다 이 값의 의미를 알 수 있게 변수로 처리해주는 게 좋을 것 같습니다.
예를 들어,
```suggestion
  const displayedRoutineCounts = 3;
  const sliceData = exercises.slice(0, displayedLoutineCounts);
```